src/main/java/bio/terra/cda/app/service/QueryService.java

```diff
@@ -28,9 +38,14 @@
 
   private static final Logger logger = LoggerFactory.getLogger(QueryService.class);
 
-  final BigQuery bigQuery = BigQueryOptions.getDefaultInstance().getService();
+  final BigQuery bigQuery = BigQueryOptions.newBuilder().setProjectId("gdc-bq-sample").build().getService();
```

Comment:
Did you mean to hard-code this?

Reply:
Yes, for now it's OK, but in the long term we'll want to create an injectable object for the bigquery service, and the project ID will be a configuration property.

Reply:
https://github.com/CancerDataAggregator/cda-service/issues/72